src/DynamicSizeList.js

```diff
@@ -324,10 +326,14 @@ const DynamicSizeList = createListComponent({
             if (mountingCorrections === 1) {
               correctScroll();
             } else {
-              if (correctionFrame) {
-                window.cancelAnimationFrame(correctionFrame);
+              if (isChrome) {
```

Comment:
Does this detect the desktop app?

Reply:
desktop app also needed to be considered as chrome so i made func to return true when it is desktop app

Reply:
Desktop app needed that animationFrame hack